Composer/packages/client/src/pages/home/styles.ts

```diff
@@ -234,19 +231,26 @@ export const mediaCardItem = {
   imageCover: css`
     ${cardItem.imageCover}
     width: 100%;
+    margin-bottom: 12px;
+    overflow: hidden;
+    display: flex;
     min-height: 95px;
     height: calc(17vw * 95 / 244);
     @media (max-width: 1416px) {
       height: calc(20vw * 95 / 244);
     }
-    margin-bottom: 12px;
-    overflow: hidden;
-    display: flex;
-    .image-cover-background {
+    @media (min-width: 2800px) {
       background: #323130;
```

Comment:
Can use a equivalent color from FluentThem.palette. If not from CommunicationColors, SharedColors

Reply:
NeutralColors.gray160

Reply:
thanks, resolved